docs/docusaurus/docs/application_integration_support.md

```diff
@@ -2,98 +2,69 @@
-⁵ Although only Airflow is supported, GX Cloud and GX OSS should work with any orchestrator that executes Python code.
+¹ We've also seen GX work with the following data sources in the past but we can't guarentee ongoing compatibility. These data sources include Clickhouse, Vertica, Dremio, Teradata, Athena, EMR Spark, AWS Glue, Microsoft Fabric, Trino, Pandas on (S3, GCS, Azure), Databricks (Spark), and Spark on (S3, GCS, Azure).<br/>
+² Support for BigQuery in GX Cloud will be available in a future release.<br/>
+³ This includes configuration storage for Expectations, Checkpoints, Validation Definitions, and Validation Result<br/>
+⁴ config_variables.yml<br/>
+⁵ Although only Airflow is supported, GX Cloud and GX OSS should work with any orchestrator that executes Python code.<br/>
 
 ### GX components
 
 The following table defines the GX components supported by GX Cloud and GX OSS.
 
-| Component    | GX Cloud                                                                                         | GX OSS                                                                       |
-|--------------|--------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|
-| Expectations | See [Available Expectations](/cloud/expectations/manage_expectations.md#available-expectations). | See [Create Expectations](/core/create_expectations/create_expectations.md). |
-| GX Agent     | All versions                                                                                     | N/A                                                                          |
-
+| Component    | GX Cloud                                                                                        | GX Core                                                               | Community                                                                  |
+| ------------ | ----------------------------------------------------------------------------------------------- | --------------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| Expectations | See [Available Expectations](/cloud/expectations/manage_expectations.md#available-expectations) | See [Expectations Gallery](https://greatexpectations.io/expectations) | See [Legacy Gallery](https://greatexpectations.io/legacy/v1/expectations/) |
+| GX Agent     | All versions                                                                                    | N/A                                                                   | N/A                                                                        |
 
-## Community-supported
+### Operating systems
 
-The following integrated applications, operating systems, and programming languages are supported by the community.
+The following table defines the operating systems supported by GX Cloud and GX OSS.
```

Comment:
As noted before: GX OSS -> GX Core.  You should use find/replace to make sure all of these instances get updated.